WalletWasabi.Daemon/Global.cs

```diff
@@ -76,7 +76,10 @@ public Global(string dataDir, string configFilePath, Config config)
 
 		BitcoinStore = new BitcoinStore(IndexStore, AllTransactionStore, mempoolService, smartHeaderChain, blocks);
 		HttpClientFactory = BuildHttpClientFactory(() => Config.GetBackendUri());
-		CoordinatorHttpClientFactory = BuildHttpClientFactory(() => Config.GetCoordinatorUri());
+		if (Config.GetCoordinatorUri() is { } uri)
```

Comment:
`Config.GetCoordinatorUri()` can be stored into a local variable instead of creating another variable later on.

`Uri? coordinatorUri = Config.GetCoordinatorUri();`

Reply:
FYI this is already what is done here. This pattern means that if `GetCoordinatorUri` is not null then store it in a variable called uri.

What I have to do here was to merge both `if`, thanks for the feedback, I hadn't noticed.